README.asciidoc

```diff
@@ -68,7 +68,7 @@ Same way you can create a test for Kubernetes, Docker, Openshift.
 | arquillian-container-configuration | To configure container configuration.                                            |
 | arquillian-create-test             | To create a test inside package.                                                 |
 | arquillian-cube-setup              | To add arquillian cube dependency & updates arquillian configuration.            |
-| arquillian-cube-create-test        | To configure a test with required Test enricher depending on selected type.      |
+| arquillian-cube-add-test           | To configure a test with required Test enricher depending on selected type.      |
```

Comment:
Yes, here changing to something like updates the test to cube fashion or something like this might be better.